kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn/src/test/filtered-resources/kjars-sources/bkm-java-model/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie.server.testing</groupId>
+    <artifactId>common-parent</artifactId>
+    <version>1.0.0.Final</version>
+  </parent>
+
+  <artifactId>bkm-java-model</artifactId>
+  <version>1.0.0.Final</version>
+  <packaging>kjar</packaging>
+
+<!-- there should be no need for Kie, Kie-DMN, etc dependencies here -->
+<dependencies>
+  <dependency>
+    <groupId>org.apache.commons</groupId>
```

Comment:
If the KJAR under test needs an external dependency, it is correct to put it in this way?

Reply:
yes, this should be exactly the same way as end users would do - declare the dependency in project's pom. One important aspect is to make it as provided if that library is already on runtime classpath e.g. kie-server.war/WEB-INF/lib

Reply:
not sure why Kie-server should need `commons-math3` or if it's actually in the dependency libs, but setting scope as provided solved the issue. Thanks as always @mswiderski !!